server/bin/gold/test-5.2.txt

```diff
@@ -976,31 +976,31 @@ run-1970-01-01T00:00:42-UTC: Processed 10 result tar balls, 7 successfully (0 pa
 1970-01-01T00:00:42.000000 DEBUG pbench-index.pbench-index main -- start processing list of tar balls
 1970-01-01T00:00:42.000000 INFO pbench-index.pbench-index main -- Starting /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/TO-INDEX/tarball-0_1970.01.01T00.42.00.tar.xz (size 212)
 1970-01-01T00:00:42.000000 DEBUG pbench-index.pbench-index main -- open tar ball
-1970-01-01T00:00:42.000000 WARNING pbench-index.pbench-index main -- Unsupported tar ball format: /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz - tar ball is missing "tarball-0_1970.01.01T00.42.00/metadata.log".
+1970-01-01T00:00:42.000000 WARNING pbench-index.pbench-index main -- Tar ball does not contain a metadata.log file: /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz - tar ball is missing "tarball-0_1970.01.01T00.42.00/metadata.log".
```

Comment:
Is there a reason for this message change?  Does it convey additional information?  It seems like it is repeating information in the message, where the old message didn't.

Reply:
No, Earlier messages were written in logger directly now we are using error code comments